model/src/main/java/org/cbioportal/genome_nexus/model/VEPInfo.java

```diff
@@ -0,0 +1,25 @@
+package org.cbioportal.genome_nexus.model;
+
+public class VEPInfo {
+
+    private final String NA_STRING = "NA";
+    private final String COMMENT = "VEP annotations are currently externally sourced from ENSEMBL. Results are subject to change without notice.";
+
+    // server corresponds to genome nexus VEP code release
+    // cache corresponds to downloaded FASTA cache
+    // comment is an additional disclaimer in the event we are using ENSEMBL VEP
+    public Version server;
+    public Version cache;
+    public String comment;
+
+    public VEPInfo(String vepServerVersion, String vepCacheVersion, String vepRegionURL) {
```

Comment:
By defining only a complex (non-default) constructor, it means that it will not be possible to construct a VEPInfo object without knowing the argument values for all three parameters here. So creating temporary / placeholder objects which get fully populated later will not be possible. Again, probably not a problem for this simple type which has only a single purpose. But in general, my concept of the "ideal" model class is a POJO with no business logic, containing only private data members, getters and setters which do direct returns / assignments, and a default constructor.

If we were to use kind of model, this business logic in this method would move into the service layer implementation, or into a service layer utility/component like "VEPInfoFactory.constructVEPInfo(String vepServerVersion, String vepCacheVersion, String vepRegionURL)". I'm not requesting that change ... just sharing the picture. One advantage of that approach is to make unit test writing easier, although in this case no unit testing is needed.